lib/ui/common/ActionSheet.dart

```diff
@@ -186,24 +208,33 @@ class _ActionSheetState extends State<ActionSheet> {
     List<Widget> listBuilder = [
       Expanded(
         child: RoundedButton(
-            viewModel: RoundedButtonViewModel(
-                title: viewModel.cancelButtonTitle,
-                onTap: () => Navigator.pop(context)),
-            style: RoundedButtonStyle.actionSheetLargeRoundedButton()),
+          viewModel: RoundedButtonViewModel(
+              title: viewModel.cancelButtonTitle,
+              onTap: () => Navigator.pop(context)),
+          style: RoundedButtonStyle.actionSheetLargeRoundedButton(),
+        ),
       )
     ];
 
     if (viewModel.confirmButtonTitle != null) {
       listBuilder.clear();
-      listBuilder.add(Expanded(
-        child: RoundedButton(
+      listBuilder.add(
+        Expanded(
+          child: RoundedButton(
             viewModel: RoundedButtonViewModel(
-                title: viewModel.cancelButtonTitle,
-                onTap: dismissActionSheet),
-            style: RoundedButtonStyle.actionSheetLargeRoundedButton()),
-      ));
-      listBuilder.add(SizedBox(width: style.buttonSpacing));
-      listBuilder.add(Expanded(child: hasSelectedItem(context, style)));
+                title: viewModel.cancelButtonTitle, onTap: dismissActionSheet),
+            style: RoundedButtonStyle.actionSheetLargeRoundedButton(),
+          ),
+        ),
+      );
+      listBuilder.add(
+        SizedBox(width: style.buttonSpacing),
+      );
+      listBuilder.add(
+        Expanded(
+          child: hasSelectedItem(context, style),
+        ),
```

Comment:
I would like to find another way to add the widgets in the list, but not sure if there's a way to do it. I will investigate a part of this PR.